File: exercicios/R/1117/1117.r
input = file('stdin','r')

cont = as.integer(0)
nota = as.double(0)
while(cont <2 ){
    n = as.double(readLines(input,n=1))

    if(n < 0.0 || n > 10.0){
        write("nota invalida",'')
    }else if(n >=0 && n <=10){
        cont = cont + 1
        nota  = nota + n
    }
}
write(sprintf("media = %.2f",nota/2),'')

File: exercicios/JavaScript/1117/1117.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n')

var count=0, soma=0;

while(true){
    var nota = parseFloat(lines.shift());

    if(count == 2){
        console.log("media = " + (soma/2).toFixed(2))
        break
    }
    if(nota > 0 && nota <=10){
        soma += nota;
        count += 1;
    }else{
        console.log("nota invalida")
    }

}